corehq/apps/app_manager/models.py

```diff
@@ -2158,6 +2158,7 @@ class CaseSearch(DocumentSchema):
     data_registry = StringProperty()
     data_registry_workflow = StringProperty()           # one of REGISTRY_WORKFLOW_*
     additional_registry_cases = StringListProperty()  # list of xpath expressions
+    expand_id_property = StringProperty()  # case property referencing another case's ID
```

Comment:
I assume we only want 1 here and not a list?

Reply:
That was my recollection from our conversation on Monday or whenever it was, but if we may ever want a list, I suppose now's the time, at least on the model, lest we need a migration later.

Though if we did support a list, what would that look like?  It'd have to be passed through a datum or datums in the suite.xml, and received by the case search backend.  That might be simplest as a space-separated list that's treated as a single string.  What do you think?

Reply:
You can just pass multiple datums with the same key. We do this for `case_type` already.

Perhaps it's best to limit it to 1 though in terms of performance and the size of the result set.